databases/week2/more-queries.sql

```diff
@@ -0,0 +1,30 @@
+-- Get all tasks assigned to users whose email ends in @spotify.com
+SELECT * 
+FROM task 
+JOIN user ON task.user_id = user.id 
+WHERE user.email LIKE '%@spotify.com';
```

Comment:
In the [provided data](https://github.com/HackYourFuture-CPH/databases/blob/main/lesson2/lesson2-data.sql) for the lesson in week 2, there is no `user_id` column in the `task` table. Instead, you have to connect via the `user_task` N:N connection table.